tests/unit/include/SugarEmailAddress/SugarEmailAddressTest.php

```diff
@@ -1329,8 +1331,6 @@ public function testGetEmailAddressWidgetEditView()
 
         self::assertTrue(is_string($result));
 
-        self::assertNotEquals($noModuleResult, $result);
```

Comment:
why did you removing tests?

Reply:
$noModuleResult does not exist. It was breaking the test framework on my local computer.

Reply:
ok